pkg/http/authorization.go

```diff
@@ -118,11 +120,25 @@ func AuthorizationMiddleware(requireOAuth bool, serverURL string, mcpServer *mcp
 	}
 }
 
+var allSignatureAlgorithms = []jose.SignatureAlgorithm{
+	jose.EdDSA,
+	jose.HS256,
```

Comment:
It looks like go-oidc does not easily support verifying symmetric tokens (the ones contain `HS` prefixes). But it is better to add this in here, in case in the future we want to support this.

Reply:
OK, I copied the list from them (IIRC)

Reply:
I copied the list from https://github.com/go-jose/go-jose/blob/3a80e136a96e747bf44049414eadc02828df4d33/jose-util/crypto.go#L48-L62

Since the original implementation was completely lenient on the signature I assumed we wanted to allow anything, hence reusing a list.